### PR TITLE
Switch to core24

### PR DESCRIPTION
**Pull request by seb128**
Update the thunderbird snap to core24

### CODE REVIEW

**Comment by seb128** [comment]
I've also published the build to the beta/core24 channel

File: snapcraft.yaml [hunk 1]
```diff
@@ -18,8 +18,7 @@ summary: Mozilla Thunderbird email application
 description: Thunderbird is a free email application that’s easy to set up and customize - and it’s loaded with great features!
 confinement: strict
 grade: stable
-base: core22
-compression: lzo
```
Comment (nteodosio) [inline snapcraft.yaml:22]: Is changing compression from lzo to xz really intended? How does it relate to switching to core24.

**Review by nteodosio — COMMENTED** [review]
Just a small question, else all is good to me.

File: snapcraft.yaml (hunk 1 above)
Comment (seb128) [reply snapcraft.yaml:22]: You are right it's not directly part of porting to core24, we had a discussion earlier this year about why libreoffice was lzo compressed despite not having that line and the (wrong?) conclusion was that the snapd default changed
(see the discussion on https://matrix.to/#/!GGqzbFAUQprdPgYYCM:ubuntu.com/$MX64gVkzlXTOAzCbnX19KxpJVDzXQEKwrmQ2qGzAe5I?via=ubuntu.com&via=matrix.org&via=envs.net)

but
```
$ sudo file /var/lib/snapd/snaps/thunderbird_492.snap 
/var/lib/snapd/snaps/thunderbird_492.snap: Squashfs filesystem, little endian, version 4.0, xz compressed, 130287863 bytes, 890 inodes, blocksize: 131072 bytes, created: Tue Jul  2 18:41:00 2024
```

so the beta/core24 build is indeed now .xz compressed... I've reverted that line deletion and force pushed, thanks for catching!

**Review by nteodosio — APPROVED** [review]

**Review by bandali — APPROVED** [review]
Overall LGTM.

**Comment by hellsworth** [comment]
These changes seem good. However, since the Thunderbird ESR is in 2 days, I propose we hold off on throwing too much change at our users all at once. I think it would make sense to let the new 128 ESR sit in stable with core22 for a couple of weeks before we update to core24. In other words, I would be fine merging this as long as it stays out of the stable channel for a couple of weeks.

**Comment by seb128** [comment]
Right, it's only targeting beta at this point. I'm going to merge since I've been using the core24 build for some time and got some positive user feedback on discourse. Let's see how it works in beta and we can consider stable later

**Pull request merged**